src/Tooltip/Tooltip.component.tsx

```diff
@@ -0,0 +1,262 @@
+    toolTipWidth: number
+) => {
+    switch (position) {
+        case 'topStart':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(0, -${toolTipHeight +
+                    tooltipMargin}px, 0)`,
+            });
+        case 'top':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(${width / 2 -
+                    toolTipWidth / 2}px, -${toolTipHeight +
+                    tooltipMargin}px, 0)`,
+            });
+        case 'topEnd':
+            return css({
+                right: 0,
+                top: 0,
+                transform: `translate3d(0, -${toolTipHeight +
+                    tooltipMargin}px, 0)`,
+            });
+        case 'rightStart':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(${width + tooltipMargin}px, 0, 0)`,
+            });
+        case 'right':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(${width + tooltipMargin}px, ${height /
+                    2 -
+                    toolTipHeight / 2}px, 0)`,
+            });
+        case 'rightEnd':
+            return css({
+                left: 0,
+                bottom: 0,
+                transform: `translate3d(${width + tooltipMargin}px, 0, 0)`,
+            });
+        case 'bottomEnd':
+            return css({
+                right: 0,
+                top: 0,
+                transform: `translate3d(0, ${height + tooltipMargin}px, 0)`,
+            });
+        case 'bottom':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(${width / 2 -
+                    toolTipWidth / 2}px, ${height + tooltipMargin}px, 0)`,
+            });
+        case 'bottomStart':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(0, ${height + tooltipMargin}px, 0)`,
+            });
+        case 'leftEnd':
+            return css({
+                left: 0,
+                bottom: 0,
+                transform: `translate3d(-${toolTipWidth +
+                    tooltipMargin}px, 0, 0)`,
+            });
+        case 'left':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(-${toolTipWidth +
+                    tooltipMargin}px, ${height / 2 - toolTipHeight / 2}px, 0)`,
+            });
+        case 'leftStart':
+            return css({
+                left: 0,
+                top: 0,
+                transform: `translate3d(-${toolTipWidth +
+                    tooltipMargin}px, 0, 0)`,
+            });
+        default:
+            return css({
+                right: 0,
+                top: 0,
+                transform: `translate3d(0, -${height + tooltipMargin}px, 0)`,
```

Comment:
Seems odd that the default positioning can't be explicitly opted into.

Reply:
It's the same as `topEnd`, for now.

Reply:
I thought it was `topEnd` at first but I looked closely and topEnd is `toolTipHeight + tooltipMargin` and this is `height + tooltipMargin`;

Maybe we should be explicit about the default and add `position = 'topEnd'` to the this.prop defaults.

Reply:
That's likely an error on my part.

Reply:
Fixed. I moved the `topEnd` case to the bottom of the switch.